deckhand/control/base.py

```diff
@@ -72,15 +77,29 @@ def req_json(self, req):
 
     def return_error(self, resp, status_code, message="", retry=False):
         resp.body = json.dumps(
-            {'type': 'error', 'message': message, 'retry': retry})
+            {'type': 'error', 'message': six.text_type(message),
+             'retry': retry})
         resp.status = status_code
 
+    def to_yaml_body(self, dict_body):
+        """Converts JSON body into YAML response body.
+
+        :dict_body: response body to be converted to YAML.
+        :returns: YAML encoding of `dict_body`.
+        """
+        if isinstance(dict_body, dict):
+            return yaml.safe_dump(dict_body)
+        elif isinstance(dict_body, list):
+            return yaml.safe_dump_all(dict_body)
+        return TypeError('Unrecognized dict_body type when converting response'
+                         ' body to YAML format.')
+
 
 class DeckhandRequestContext(object):
 
     def __init__(self):
         self.user = None
-        self.roles = ['anyone']
+        self.roles = ['*']
```

Comment:
I'm going to change it to an empty list. Since Deckhand currently doesn't have RBAC implemented this technically doesn't matter right now. I changed it to '*' for whatever reason, don't recall why.